spark/src/test/scala/com/gu/tableversions/spark/VersionedDatasetSpec.scala

```diff
@@ -205,29 +198,6 @@ class VersionedDatasetSpec extends FlatSpec with Matchers with SparkHiveSuite {
       .parquet(path.toString)
       .as[T]
 
-  class StubTableVersions(
-      currentVersions: Map[TableName, TableVersion],
-      committedTableUpdatesRef: Ref[IO, List[(TableName, TableVersions.TableUpdate)]])
-      extends TableVersions[IO] {
-
-    def committedTableUpdates: IO[List[(TableName, TableVersions.TableUpdate)]] =
-      committedTableUpdatesRef.get
-
-    override def init(
-        table: TableName,
-        isSnapshot: Boolean,
-        userId: UserId,
-        message: UpdateMessage,
-        timestamp: Instant): IO[Unit] =
-      IO.unit
-
-    override def currentVersion(table: TableName): IO[TableVersion] =
-      IO(currentVersions(table))
-
-    override def commit(table: TableName, newVersion: TableVersions.TableUpdate): IO[TableVersions.CommitResult] =
-      committedTableUpdatesRef.update(_ :+ table -> newVersion).map(_ => CommitResult.SuccessfulCommit)
-  }
-
```

Comment:
After refactoring, the `InMemoryTableVersions` class is so simple that the stub isn't much simpler - hence we're ditching this stub and just using the `InMemoryTableVersions` implementation instead in these tests.